js/clickup-add-issues/clickupClient.ts

```diff
@@ -3,12 +3,24 @@ import { request } from "https";
 import { ClientRequest } from "http";
 import * as cuSettings from "./cudata.json";
 
+
+type stringToValue = {
+    [key: string]: string[]; 
+};
+
 type CuRequestType = {
     name: string;
     description: string;
+    source: string;
     tags: Array<string>;
 }
 
+
+const tagsMap: stringToValue = {
+    "transformhub": [],
+    "cloud-platform-panel" : ["front", "panel"],
```

Comment:
The mapping is good, but could we move it outside the code, to a configuration file?
Then, if someone wants to modify it, they don't need to build the Sequence anew and we could also showcase it as a general code in our Samples.